spec/lib/xclisten_spec.rb

```diff
@@ -22,12 +22,14 @@ def run(args)
       let(:devices) { XCListen::IOS_DEVICES }
 
       it "has ALL of them" do
-        devices['iphone5s'].should == 'iPhone Retina (4-inch 64-bit)'
-        devices['iphone5'].should == 'iPhone Retina (4-inch)'
-        devices['iphone4'].should == 'iPhone Retina (3.5-inch)'
-        devices['ipad2'].should == 'iPad'
-        devices['ipad4'].should == 'iPad Retina'
-        devices['ipad_air'].should == 'iPad Retina (64-bit)'
+        devices['iphone6_plus'].should == 'iPhone 6 Plus'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.